.babelrc

```diff
@@ -1,7 +1,9 @@
 {
   "presets": [
-    "react",
-    "stage-3",
+    "@babel/env",
```

Comment:
Perhaps move this line down to line 7, in place of `env` , next to the nested `targets` declaration?